common/djangoapps/util/milestones_helpers.py

```diff
@@ -150,6 +151,31 @@ def fulfill_course_milestone(course_key, user):
             add_user_milestone({'id': user.id}, milestone)
 
 
+def get_required_content(course, user):
```

Comment:
@mattdrayer i have moved this method into milestones_helper file for better usability.